webdev/flareact/config.json

```
@@ -9,7 +9,7 @@
         "cmd": "",
         "env": "./azion/webdev.env",
         "output-ctrl": "on-error",
-        "default": "npx --yes azion-framework-adapter@0.2.3 build --config ./azion/kv.json || exit $? ;"
+        "default": "[[ $(node -v |tr -dc '0-9' | cut -c1-2) -gt 16 ]] && OPENSSL_LEGACY=--openssl-legacy-provider; NODE_OPTIONS=$OPENSSL_LEGACY; npx --yes azion-framework-adapter@0.2.3 build --config ./azion/kv.json || exit $? ;"
```

Couldn't it be written like that to make it shorter?

```suggestion
        "default": "[[ $(node -v |tr -dc '0-9' | cut -c1-2) -gt 16 ]] && NODE_OPTIONS=--openssl-legacy-provider"; npx --yes azion-framework-adapter@0.2.3 build --config ./azion/kv.json || exit $? ;"
```

---

In that case I would need to use "export". I preferred to avoid this to not set a variable in the user's session.